vuepress/docs/next/tutorials/devops/change-default-datasource.md

```diff
@@ -10,22 +10,22 @@ This tutorial explains how to customize our EAP and Wildfly base Docker images,
 
 -   Keycloak and the Entando Cluster Infrastructure have been deployed
 
--   The required external databases and other services are up and running.
+-   The required external databases and other services are up and running
 
--   Your own custom project as described in [the tutorial](./build-core-image.md) 
+-   Your own custom project as described in [this tutorial](./build-core-image.md) 
 
 ## Steps
 
 ### 1. Create a Docker project for your own base image to Entando's standard base image(s)
-Create a project, ideally in Git repository, and put the following files in the folder
+Create a project, ideally in Git a repository, and put the following files in the folder.
 #### 1.1. The standalone.xml/standalone-openshift.xml file of choice 
 It is essential for your custom Docker image to use a file from one of the official Entando Docker base images.
 
-If you intend to use provide your own EAP based image, please use this 
+If you intend to provide your own EAP base image, please use this 
```

Comment:
don't think this is right anymore, EAP-based image is not EAP base image

Reply:
this confused me! i'm going to bring it up in standup. base and based are used in this doc and idk which one is accurate and didn't take the time to dive into it. first i had EAP-based, but then looking at other sections it seemed like they meant EAP base - a base image created with EAP - so went that way...